### DISCUSSION TITLE
CPU Mutlithread detection

### DISCUSSION THREAD

**Discussion by struthio** [AlexeyAB/darknet#8760] [Q&A]
hi,

I was running darknet_images.py on RPi (so CPU detection with OpenCV).
Although when script is running I noticed that only 1 code (out of 4) is used for detection.
Is there any way to run detection on 4 cores to speedup process ?

**Comment by stephanecharette** [comment]
Modify your code to load 4 instances. Or load images on 1 thread, and run
inference on the other 3?

**Reply by struthio** [reply]
RPi have limited RAM, so 4 images concurrently might be too much.
I was asking of running detection on 1 image to run in 4 threads.
